src/webui/prefjson.cpp

```diff
@@ -114,6 +114,7 @@ QByteArray prefjson::getPreferences()
     data["ip_filter_enabled"] = session->isIPFilteringEnabled();
     data["ip_filter_path"] = Utils::Fs::toNativePath(session->IPFilterFile());
     data["ip_filter_trackers"] = session->isTrackerFilteringEnabled();
+    data["banned_ip_list"] = session->getBannedIPList().join("\n");
```

Comment:
Agree with you: "list" is unneeded in `getBannedIPList()` because the function signature encodes this piece of information (in the return type). The same is true for the setter. Moreover, this project does not use "get" prefix. Hence I would make function signatures as follows:
`QStringList bannedIPs() const;
void setBannedIPs(const QStringList &ips);
`